app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,44 @@
+class AnimalsController < ApplicationController
+    def index
+        animals = Animal.all
+        render json:animals
+    end
```

Comment:
Step 2 of the Tracker was to be able to display all the animals in the database. It looks that is here and conveys that in an efficient method. Variables names make sense, there is no psuedo-code however the class name and content make it apparent as to what this represents.